modules/lti/src/main/resources/tools/series/series.js

```diff
@@ -115,6 +121,32 @@ function loadPage(page) {
         }
       }
 
+      // Download buttons
+      // Create a button for each track that contains a link to a mp4 or webm
+      var buttonData = [];
+      var tracks = episode.mediapackage.media.track;
+      tracks = Array.isArray(tracks) ? tracks : [tracks];
+      for (var k = 0; k < tracks.length; k++)
+      {
+        if( tracks[k].url.endsWith('mp4') || tracks[0].url.endsWith('webm')) {
+
+          // Calculate filesize
+          var fileSize = (tracks[k].video.bitrate / 8 / 1000 / 1000)      // Converts to byte, kilobyte, megabyte /s
+            * (tracks[k].video.framecount / tracks[k].video.framerate);
```

Comment:
Is there no way to get access to the actual file size? This seems like a ... hacky approximation?

Reply:
This also disregards the audio.
If the size is not in the JSON, it probably makes sense to not display it.
And I know that I originally suggested to show the size :D

Reply:
It may seem that way because it is ^^'. The JSON containing the track information does not specify a file size, so I thought I'd use the other information at hand. Not sure if I can get the file size from the url itself, something to look into.

Reply:
So apparently you can just _use a HTTP HEAD request and then look at Content-Length response header. ( [https://bitexperts.com/Question/Detail/3316/determine-file-size-in-javascript-without-downloading-a-file](url) )

Unsure whether it is worth it to send a bunch of web requests, since the hacky approximation seems to arrive at fairly similar numbers.

Reply:
> If the size is not in the JSON, it probably makes sense to not display it.

I guess I agree with that. Doesn't seem like a critical feature, so just remove the file size?

Reply:
Huh, I figured the bitrate would include the audio. 
But if showing file size is not that important anyway, I'll cut it.